apps/segfault-on-filtered-vector-search/requirements.txt

```diff
@@ -0,0 +1,4 @@
+weaviate-client==3.6.0
```

Comment:
use the current one? You could also do `weaviate-client>=3.8.0 ` so it always picks the latest

Reply:
Good point, I'll update.